app/services/TtpService.scala

```diff
@@ -142,20 +146,22 @@ class TtpService @Inject() (
     val directDebitDetails = journey.fold(_.directDebitDetails, _.directDebitDetails)
     val selectedPaymentPlan = journey.fold(_.selectedPaymentPlan, _.selectedPaymentPlan)
     val correlationId = journey.fold(_.correlationId, _.correlationId)
+    val regimeDigitalCorrespondence = journey.fold(_.eligibilityCheckResult.regimeDigitalCorrespondence, _.eligibilityCheckResult.regimeDigitalCorrespondence)
+    val customerDetail = journey.fold(_.eligibilityCheckResult.customerDetails, deriveCustomerDetail)
```

Comment:
since this and `regimeDigitalCorrespondence` are being added now, it'd be handy to add unit tests testing that the correct values are populated in the request - especially around the email and the email source and toLowerCase stuff

Reply:
sure thing